atomic_reactor/schemas/config.json

```diff
@@ -533,7 +533,7 @@
         "type": "string"
     },
     "skip_koji_check_for_base_image": {
-        "description": "Allow base image without related koji build (insecure)",
+        "description": "Allow base image without related koji build (insecure), assuming NVR isn't found in Dockerfile, if NVR is found it Dockerfile koji check will be performed regardless",
```

Comment:
`if NVR is found it Dockerfile koji`  IMHO `it` is extra there.

Why Dockerfile? Aren't image labels inspected?

Reply:
fixed